src/main/java/com/revature/byteshare/tags/Tag.java

```diff
@@ -17,8 +18,8 @@ public class Tag {
     private int tag_id;
 
     @ManyToOne
-    @JoinColumn(name = "recipe_Id") //TODO make sure this is the correct column string
-    private int recipe_id;
+    @JoinColumn(name = "recipe_Id", nullable = false) //TODO make sure this is the correct column string
```

Comment:
Probably make this recipe_id? Don't think itll cause any problems but just in case it will down the line